source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,204 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+You can change documents in a MongoDB collection using **update** and
+**replace** operations. Update operations modify specified fields in one
+or more documents while leaving other fields and values unchanged.
+Replace operations remove all existing fields in one or more
+documents and substitute them with new fields and values.
```

Comment:
the antecedent to `them` is slightly ambiguous. Do replace operations substitute fields? documents?